bigtable/client/detail/bulk_mutator.cc

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/detail/bulk_mutator.h"
```

Comment:
You're using `#include "..."` here but `#include <...>` a few lines below. Since you're using full-path `#include`s for all of these, it's a bit confusing as to when to use which.

Are public headers using `#include <...>` and private headers use `#include "..."`? Or another pattern?

Reply:
See above.  I am fixing any inconsistencies I see, but not planning to fix them all in this PR.